docs/src/components/productX/XDateRangeDemo.tsx

```diff
@@ -105,7 +120,7 @@ export default function XDateRangeDemo() {
               '& > div': {
                 borderRadius: 1,
                 overflow: 'auto',
-                bgcolor: '#fff',
+                bgcolor: '#FFF',
```

Comment:
convention, and simplify the diff

```suggestion
                bgcolor: '#fff',
```

Reply:
I thought the convention was the opposite: hex codes all in uppercase (e.g., look at the branding theme file)  😅 Do we have these conventions documented somewhere?